chart/templates/secrets/metadata-connection-secret.yaml

```diff
@@ -31,6 +31,8 @@ apiVersion: v1
 metadata:
   name: {{ .Release.Name }}-airflow-metadata
   labels:
+    tier: airflow
+    component: connection
```

Comment:
I'm considering not setting a component for some of these, like this one. Thoughts on having it on "most but not all"?

Reply:
Yeah, this one feels wrong, especially as it used by all "components"